src/components/Editor/helpers/setZoomEvent.ts

```diff
@@ -18,20 +18,37 @@ export const setZoomEvent = (
     const event = e as unknown as WheelEvent
     if (!(event.ctrlKey || event.metaKey)) return
 
-    event.preventDefault()
-
-    let fontSize = parseFloat(editorElement.dom.style.fontSize)
-    if (isNaN(fontSize)) {
-      fontSize = storedFontSize
-    }
+    changeFontSize(event, editorElement, storedFontSize, event.deltaY < 0)
+  })
 
-    if (event.deltaY < 0) {
-      fontSize = Math.min(fontSize + 1, maxFontSize)
-    } else {
-      fontSize = Math.max(fontSize - 1, minFontSize)
-    }
+  element.addEventListener('keypress', (e) => {
+    const event = e as unknown as KeyboardEvent
+    if (!(event.ctrlKey || event.metaKey)) return
+    if (event.key !== '+' && event.key !== '-') return
```

Comment:
I cannot zoom when I try pressing `ctrl` + `=` (usually the key on the left side of the backspace key). I know that's not part of the requirement, but could you add that functionality as well? Since many users would expect to zoom in using that key.

Reply:
added :+1: